Makefile

```
@@ -81,8 +81,11 @@ lint: images
 lint-errors: images
 	@$(DOCKER) run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos8 bash -c "scripts/run_lint.sh --errors-only"
 
+tests7: images
+	@$(DOCKER) run --rm --user root -v $(shell pwd):/data:Z $(IMAGE)/centos8 pytest --show-capture=$(SHOW_CAPTURE)
+
 tests8: images
-	@$(DOCKER) run --rm -v $(shell pwd):/data:Z $(IMAGE)/centos8 pytest --show-capture=$(SHOW_CAPTURE)
+	@$(DOCKER) run --rm --user root -v $(shell pwd):/data:Z $(IMAGE)/centos8 pytest --show-capture=$(SHOW_CAPTURE)
```

I'd use what we have in `tests` already `--user=$(id -ur):$(id -gr)`.
```suggestion
	@$(DOCKER) run --rm --user=$(id -ur):$(id -gr) -v $(shell pwd):/data:Z $(IMAGE)/centos8 pytest --show-capture=$(SHOW_CAPTURE)
```

---

That's a better solution, agreed! Didn't even see the `tests` command